packages/test/test-gc-sweep-tests/src/ignoreErrorLogger.ts

```diff
@@ -16,7 +18,7 @@ export class IgnoreErrorLogger extends EventAndErrorTrackingLogger {
     public readonly errorEvents: ITelemetryBaseEvent[] = [];
     public readonly errorEventStats: { [key: string]: number; } = {};
 
-    send(event: ITelemetryBaseEvent): void {
+    public send(event: ITelemetryBaseEvent): void {
         this.events.push(event);
```

Comment:
The out of memory problem will still happen here because all the events are still stored in memory, right?

Reply:
If there's only 200,000 ops being performed, we don't run out of memory. It's quite interesting, I was running the standard summarizer logic. Granted it only ran for an hour.

Reply:
I figure I'd add custom logic to constantly write to the file later, this is more of a v1, would you prefer it be in this PR?

Reply:
We only run out of memory when the number of datastores increases at a rapid rate.

Reply:
It's fine to have this for v1 as long as the tests are not running out of memory. Just keep in mind that as you increase the test time, this will increase the memory usage.
Can you create an issue to do this later (v2).

Reply:
I created AB#1936 for this.